### PR TITLE
De-fork aws-sdk-go

### PR DESCRIPTION
**Pull request by sayboras**
### Description

As discussed in slack before, based on my understanding, there is no real benefit to use forked version of aws-sdk-go in weaveworks org. Moreover, this repo is publicly available, so will not help with any development activities for coming feature.

### Approach

Remove the replace directive statement, run `go mod tidy` to clean up go.sum as well.

### Testing

<details>
<summary>Create cluster</summary>

```shell script
$ ./eksctl create cluster --managed --node-private-networking
[ℹ]  eksctl version 0.19.0-dev+c463a1cc.2020-04-22T21:36:01Z
[ℹ]  using region us-east-2
[ℹ]  setting availability zones to [us-east-2a us-east-2c us-east-2b]
[ℹ]  subnets for us-east-2a - public:192.168.0.0/19 private:192.168.96.0/19
[ℹ]  subnets for us-east-2c - public:192.168.32.0/19 private:192.168.128.0/19
[ℹ]  subnets for us-east-2b - public:192.168.64.0/19 private:192.168.160.0/19
[ℹ]  using Kubernetes version 1.15
[ℹ]  creating EKS cluster "amazing-party-1587555385" in "us-east-2" region with managed nodes
[ℹ]  will create 2 separate CloudFormation stacks for cluster itself and the initial managed nodegroup
[ℹ]  if you encounter any issues, check CloudFormation console or try 'eksctl utils describe-stacks --region=us-east-2 --cluster=amazing-party-1587555385'
[ℹ]  CloudWatch logging will not be enabled for cluster "amazing-party-1587555385" in "us-east-2"
[ℹ]  you can enable it with 'eksctl utils update-cluster-logging --region=us-east-2 --cluster=amazing-party-1587555385'
[ℹ]  Kubernetes API endpoint access will use default of {publicAccess=true, privateAccess=false} for cluster "amazing-party-1587555385" in "us-east-2"
[ℹ]  2 sequential tasks: { create cluster control plane "amazing-party-1587555385", create managed nodegroup "ng-0185ad98" }
[ℹ]  building cluster stack "eksctl-amazing-party-1587555385-cluster"
[ℹ]  deploying stack "eksctl-amazing-party-1587555385-cluster"
[ℹ]  building managed nodegroup stack "eksctl-amazing-party-1587555385-nodegroup-ng-0185ad98"
[ℹ]  deploying stack "eksctl-amazing-party-1587555385-nodegroup-ng-0185ad98"
[✔]  all EKS cluster resources for "amazing-party-1587555385" have been created
[✔]  saved kubeconfig as "/home/tammach/.kube/config"
[ℹ]  nodegroup "ng-0185ad98" has 2 node(s)
[ℹ]  node "ip-192-168-134-163.us-east-2.compute.internal" is ready
[ℹ]  node "ip-192-168-187-92.us-east-2.compute.internal" is ready
[ℹ]  waiting for at least 2 node(s) to become ready in "ng-0185ad98"
[ℹ]  nodegroup "ng-0185ad98" has 2 node(s)
[ℹ]  node "ip-192-168-134-163.us-east-2.compute.internal" is ready
[ℹ]  node "ip-192-168-187-92.us-east-2.compute.internal" is ready
[ℹ]  kubectl command should work with "/home/tammach/.kube/config", try 'kubectl get nodes'
[✔]  EKS cluster "amazing-party-1587555385" in "us-east-2" region is ready

$ kgno                                                                                      
NAME                                            STATUS   ROLES    AGE    VERSION
ip-192-168-134-163.us-east-2.compute.internal   Ready    <none>   5m4s   v1.15.10-eks-bac369
ip-192-168-187-92.us-east-2.compute.internal    Ready    <none>   5m5s   v1.15.10-eks-bac369

```

</details>

### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [x] Manually tested
- [ ] Added labels for change area (e.g. `area/nodegroup`), target version (e.g. `version/0.12.0`) and kind (e.g. `kind/improvement`)
- [ ] Make sure the title of the PR is a good description that can go into the release notes

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -66,7 +65,6 @@ replace (
 	// Override since git.apache.org is down.  The docs say to fetch from github.
 	git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v14.0.0+incompatible
-	github.com/aws/aws-sdk-go => github.com/weaveworks/aws-sdk-go v1.25.14-0.20191218135223-757eeed07291
```
Comment (martina-if) [inline go.mod:69]: Let's leave this line here but commented out in case we need it in the future

Comment (sayboras) [reply go.mod:69]: :100:

File: go.mod [hunk 2]
```diff
@@ -66,7 +65,7 @@ replace (
...
+	//github.com/aws/aws-sdk-go => github.com/weaveworks/aws-sdk-go v1.25.14-0.20191218135223-757eeed07291
```
Comment (cPu1) [inline go.mod:68]: Minor nit: let's leave a space after the comment to make it more readable.
```suggestion
	// github.com/aws/aws-sdk-go => github.com/weaveworks/aws-sdk-go v1.25.14-0.20191218135223-757eeed07291
```

Comment (sayboras) [reply go.mod]: Good spot :100:

**Pull request merged**